src/bot.ts

```diff
@@ -2,13 +2,12 @@
  * Deepindigo
  *
  */
-const { MOVE, SWITCH } = require('leftovers-again/src/decisions');
-
+import { PokemonData, State, MOVE, SWITCH, AI } from 'leftovers-again';
```

Comment:
This is a little funky because I may be importing the type, or importing the class itself, I got confused when I was playing around with it

Reply:
Related, we may have to update leftovers-again to export some of these if it isn't already